src/components/Attachment/Attachment.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable sort-keys */
```

Comment:
I guess this is dev-only leftover. Shall we remove it?

Reply:
I moved it to affected lines, I'd love to disable the rule altogether though.